fastly/resource_fastly_service_acl_entries_test.go

```diff
@@ -102,6 +102,49 @@ func TestAccFastlyServiceAclEntries_create(t *testing.T) {
 	})
 }
 
+func TestAccFastlyServiceAclEntries_create_more_than_one_page(t *testing.T) {
+	var service gofastly.ServiceDetail
+	serviceName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+	aclName := fmt.Sprintf("ACL %s", acctest.RandString(10))
+
+	defaultPerPage := 100
+	var expectedRemoteEntries []map[string]any
+	for i := 1; i <= defaultPerPage*2; i++ {
+		entry := map[string]any{
+			"id":      "",
+			"ip":      fmt.Sprintf("%d.0.0.1", i),
+			"subnet":  "24",
+			"negated": false,
+			"comment": "ACL Entry 1",
+		}
+		expectedRemoteEntries = append(expectedRemoteEntries, entry)
+	}
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		ProviderFactories: testAccProviders,
+		CheckDestroy:      testAccCheckServiceVCLDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccServiceACLEntriesConfigOneACLWithEntries(serviceName, aclName, expectedRemoteEntries, true),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServiceExists("fastly_service_vcl.foo", &service),
+					testAccCheckFastlyServiceACLEntriesRemoteState(&service, serviceName, aclName, expectedRemoteEntries),
+					resource.TestCheckResourceAttr("fastly_service_acl_entries.entries", "entry.#", "1"),
```

Comment:
This is failing with...

```
resource_fastly_service_acl_entries_test.go:123: 
Step 1/2 error: Check failed: 
Check 3/3 error: fastly_service_acl_entries.entries: Attribute 'entry.#' expected "1", got "200"
```

Now that the issue with reading ACL entries is fixed, this needs to be updated to represent the 200 entries being returned...

```suggestion
					resource.TestCheckResourceAttr("fastly_service_acl_entries.entries", "entry.#", "200"),
```